api/helpers.go

```diff
@@ -51,6 +52,33 @@ func (a *API) sendTx(tx []byte) (*cometcoretypes.ResultBroadcastTx, error) {
 	return resp, nil
 }
 
+func (a *API) getBlockFromAppOrIndexer(height int64) (*Block, error) {
```

Comment:
why is it useful to query either the app or the indexer? wouldn't it be enough to always just query one? if this is necessary at least I would document it

Reply:
good point, maybe we could refactor this, but for now i just added a comment

Reply:
actually, in the end, i split up the crucial hotfix (#1339) and removed this "getBlockFromAppOrIndexer" completely